components/Image.jsx

```diff
@@ -1,4 +1,4 @@
-import Image from 'next/image'
+import Image from "next/legacy/image";
```

Comment:
This import was changed automatically by `npx @next/codemod next-image-to-legacy-image .`
    
See https://nextjs.org/docs/messages/next-image-upgrade-to-13

Reply:
can we run npx `@next/codemod next-image-experimental .` too?

Reply:
No, someone can do that in another PR. The documentation says:

> Please note this second codemod is experimental and only covers static usage (such as <Image src={img} layout="responsive" />) but not dynamic usage (such as <Image {...props} />).